x-pack/plugins/security_solution/public/common/links/app_links.ts

```diff
@@ -30,17 +32,27 @@ export const links = Object.freeze([
 
 export const getFilteredLinks = async (
   core: CoreStart,
-  plugins: StartPlugins
+  plugins: StartPlugins,
+  experimentalFeatures: Readonly<ExperimentalFeatures>
 ): Promise<AppLinkItems> => {
   const managementFilteredLinks = await getManagementFilteredLinks(core, plugins);
 
+  const threatHuntingFilteredLinks = {
```

Comment:
I hesitated to go the opposite way (adding the threatIntelligence entry to the threatHuntingLandingLinks.link property) but going this current route meant modifying one less file.

Reply:
no need to do this, links have support for experimentalFeatures, you can add the `experimentalKey: 'threatIntelligenceEnabled',` prop directly to the threat intelligence link definition at: https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/threat_intelligence/links.ts

Reply:
hahaha I had took a not of this `experimentalKey` while working on another feature, tried it and didn't get it to work... I'll try again thank you :)

Reply:
yeah that worked, I don't know what I did... must have forgotten to update my kibana.yml... 🤦‍♂️